extensionsGallery.json

```diff
@@ -5278,7 +5339,7 @@
 					"metadataItems": [
 						{
 							"name": "TotalCount",
-							"count": 86
+							"count": 87
```

Comment:
Update the insiders gallery count too please 😄

Reply:
Oops, forgot to git add 🙃